### ISSUE TITLE
Filebot CLI ambiguous error  (Ubuntu)

### ISSUE THREAD

**Issue by nonissue** [filebot/filebot#18]
I greatly appreciate the often thankless work you (@rednoah) do. Filebot has saved me dozens of hours (if not more), and is obviously a highly valued utility for thousands of people. 

That being said, Filebot just stopped working on a remote headless ubuntu server I manage (purely for friends) without anything changing on the server. I understand I am responsible for keeping software up to date, and I'm not asking you to support dozens of versions. It would be nice, though, to have some kind of error message appear as all I got were the following:

```
$ filebot -script fn:sysinfo
Script not found: sysinfo
java.io.FileNotFoundException: Script not found: sysinfo
	at net.filebot.cli.ScriptBundle.getScript(ScriptBundle.java:54)
	at net.filebot.cli.ScriptShell.runScript(ScriptShell.java:72)
	at net.filebot.cli.ArgumentProcessor.runScript(ArgumentProcessor.java:113)
	at net.filebot.cli.ArgumentProcessor.run(ArgumentProcessor.java:28)
	at net.filebot.Main.main(Main.java:124)
Failure (°_°)
```

and 

```
$ filebot -script fn:amc
Script not found: amc
java.io.FileNotFoundException: Script not found: amc
	at net.filebot.cli.ScriptBundle.getScript(ScriptBundle.java:54)
	at net.filebot.cli.ScriptShell.runScript(ScriptShell.java:72)
	at net.filebot.cli.ArgumentProcessor.runScript(ArgumentProcessor.java:113)
	at net.filebot.cli.ArgumentProcessor.run(ArgumentProcessor.java:28)
	at net.filebot.Main.main(Main.java:124)
Failure (°_°)
```

Though not at all your fault, I had to spend a little while figuring out the source of the errors as Filebot does not update through the ubuntu package manager (as far as I could tell). 

The fix was of course to download the latest update. 

TL;DR: It would be nice if the error message thrown by the script in these situations was more descriptive about the solution. Thank you again for all your hard work.

**Comment by rednoah** [comment]
1.
If you install the `deb` package manually then you'll need to update manually. If you install the `snap` package from the ubuntu developer repository then it'll auto-update, when you ask it to update everything at the very least.

2.
Unfortunately, I can't make error messages better retroactively, when something unexpected happens now that I didn't expect 1-2 years ago. You're probably using an old version that uses an old script jar that got lost when syncing the data files.

**Comment by nonissue** [comment]
1. Ah, I think part of the issue that I'm running Ubuntu 15.04 and so as far as I could figure out I had to install manually (again, not your problem).

2. I was running 4.7.2 (I'm pretty sure) when I ran in to the problem. I'm not sure if that matters.

I understand why you closed this issue. For an OSS project of this size, supporting so many deployments and operating systems must be a nightmare, especially while maintaining a robust GUI app and a fully-featured CLI app. Hopefully if anyone else runs in to the same problem, they'll stumble upon this GitHub issue. Thanks for your work and the quick response.

**Comment by rednoah** [comment]
`Ubuntu 15.04` already have `snap` support? I'd upgrade to the latest LTS at least. I think I did break `4.7.2` accidentally at some point. Sorry about that.
